packages/peregrine/lib/talons/SearchBar/useSearchBar.js

```diff
@@ -6,10 +6,10 @@ import { useDropdown } from '../../hooks/useDropdown';
 const initialValues = { search_query: '' };
 
 export const useSearchBar = () => {
-    const [valid, setValid] = useState(false);
-    const { elementRef, expanded, setExpanded } = useDropdown();
     const history = useHistory();
     const { push } = history;
+    const [valid, setValid] = useState(false);
+    const { elementRef, expanded, setExpanded } = useDropdown();
```

Comment:
Is there some meaning in this change that I'm missing?

Reply:
Just how it rebased I guess.

Reply:
Cool, reverted it. This is approved now.